build.gradle

```diff
@@ -111,3 +121,32 @@ aemCompose {
     includeProjects ':app:*'
     includeProjects ':content:*'
 }
+
+fork {
+    config {
+        copyFile()
+        moveFile([
+                "/example": "/{{projectName}}"
```

Comment:
this variables are currently injected from `fork.properties` file but user will be asked by interactive GUI window built using SwingBuilder :>